memcached.c

```diff
@@ -9294,12 +9297,28 @@ int main (int argc, char **argv) {
             }
             free(subopts_orig);
             break;
+    case 'N':
+        settings.num_napi_ids = atoi(optarg);
+        if (settings.num_napi_ids <= 0) {
+	    fprintf(stderr, "Number of NAPI_IDs must be greater than 0\n");
+	    return 1;
+        }
+        break;
         default:
             fprintf(stderr, "Illegal argument \"%c\"\n", c);
             return 1;
         }
     }
 
+    if (settings.num_napi_ids && settings.num_threads) {
+        if (settings.num_napi_ids > settings.num_threads) {
+            fprintf(stderr, "Number of NAPI_IDs(%d) cannot be more than num_threads (%d)\n",
```

Comment:
The actual option is called --threads (-t). should also say something like "number supplied to --napi-ids"

Reply:
Agree, will fix it in next rev.